packages/charts/examples/components/demo/chart-types/spark/chart-docs-spark.component.html

```diff
@@ -115,6 +115,19 @@ <h3>update vs. updateSparks</h3>
     <nui-spark-chart-multiple-example></nui-spark-chart-multiple-example>
 </nui-example-wrapper>
 
+<h2>Area with Multiple Series</h2>
+<p>This is very similar to the above example with two major changes. Here we use the
+    <code><a href="../classes/AreaRenderer.html" target="_blank">AreaRenderer</a></code> and the
+    <code><a href="../miscellaneous/functions.html#stackedAreaAccessors" target="_blank">stackedAreaAccessors</a></code> 
+    function to generate our accessors and property set the <code>x</code>, <code>y</code>, <code>y0</code> and <code>y1</code> properties. Check out the 
+    <a href="../additional-documentation/chart-types/area.html" target="_blank">Area Chart </a>
```

Comment:
For links from one "additional documentation" page to another "additional documentation" page, we've been using `target="_parent"`, so that the new page opens in the same tab. The API links above with the `_blank` target are correct because in that case we want to open a new tab.

```suggestion
    <a href="../additional-documentation/chart-types/area.html" target="_parent">Area Chart </a>
```